src/RestWrite.js

```diff
@@ -27,6 +27,9 @@ function RestWrite(config, auth, className, query, data, originalData) {
   this.auth = auth;
   this.className = className;
   this.storage = {};
+  this.runOptions = {
+    acl:['*']
```

Comment:
No, the acl options shouldn't be set by default.  A master key query shouldn't have an acl key set, for example.. See `ExportAdapter.update`